packages/fast-components-react-msft/app/app.tsx

```diff
@@ -16,11 +16,14 @@ import Site, {
     SiteMenuItem,
     SiteTitle,
     SiteTitleBrand,
+    ShellSlot
 } from "@microsoft/fast-development-site-react";
 import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
 import { Direction } from "@microsoft/fast-application-utilities";
 import * as examples from "./examples";
 import Hypertext from "../src/hypertext";
+import ColorPicker, { IColorConfig } from "./color-picker";
```

Comment:
nit: In the past we would alphabetize import statements.  Is this still a standard we want to follow?

Reply:
We alphabetize inside a statement but I don't think we're alphabetizing the statements themselves. To me, there is a real benefit to that.